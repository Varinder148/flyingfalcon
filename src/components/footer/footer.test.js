import { shallow } from "enzyme";
import Footer from "./footer.component";

describe("Footer tests", () => {
  it("checks if footer is correct", () => {
    const wrapper = shallow(<Footer />);
    expect(wrapper).toMatchSnapshot();
  });
});
