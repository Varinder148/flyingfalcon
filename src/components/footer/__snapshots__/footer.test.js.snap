// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer tests checks if footer is correct 1`] = `ShallowWrapper {}`;
